tasks/main.yml

```diff
@@ -1,8 +1,7 @@
 ---
 
 - name: install ES dependencies
-  apt: pkg={{ item }} state=present update_cache=true cache_valid_time=3600
-  with_items: [openjdk-7-jre-headless]
+  apt: pkg=openjdk-{{ ansible_distribution_version|get_openjdk_version }}-jre-headless state=present update_cache=true cache_valid_time=3600
```

Comment:
Ah thats brilliant! I didn't like having to add another function just for that. I'll update now, thanks @pkatseas !